tests/unit/models/container/test_model_service.py

```
@@ -0,0 +1,189 @@
+        assert service.metadata == metadata
+        assert service.health_status == "healthy"
+
+    def test_frozen_model(self):
+        """Test that model is frozen (immutable)."""
+        service = ModelService(
+            service_id=uuid4(),
+            service_name="test",
+            service_type="compute",
+        )
+
+        with pytest.raises(Exception):  # ValidationError for frozen model
```

Ruff (0.14.1)

Do not assert blind exception: `Exception`

(B017)